### DISCUSSION TITLE
폭이 좁은 페이지(예: 태블릿, 또는 모바일)에서 스크롤 미발생시 사이드바 메뉴 호출 불가

### DISCUSSION THREAD

**Discussion by julyrunt** [pronist/hELLO#176] [Ideas]
**환경 (버그가 발생한 환경을 적어주세요.):**

- 운영체제: windows
- 브라우저: chrome
- 버전: 109.0.5414.75(공식 빌드) (64비트)
- 스킨 버전 : 4.0.0

**설명**
폭이 좁은 페이지(예: 태블릿, 또는 모바일)에서 스크롤이 미발생되는 페이지 방문시 해당 페이지에서 다른 페이지로 이동하기 위해서 사이드바 메뉴를 호출할 수 없습니다.

**발생 과정**
폭이 좁은 모바일 기기 레이아웃에서 스크롤시 출력되는 상단바에 사이드바 메뉴 호출 버튼이 종속되어 있기 때문에,
스크롤이 발생 할 수 없는 높이의 컨텐츠 영역을 지닌 페이지에서는 사이드바를 호출 할 수 없습니다. 

상단바를 항상 출력하게 하거나, 숨어선 안될 사이드바 메뉴 버튼 등을 분리하는 것이 필요해 보입니다.

**스크린샷 및 링크**
![스크린샷 2023-01-24 14 34 25](https://user-images.githubusercontent.com/107601777/214219721-27377ba5-680c-44a4-b315-0f5233229d37.png)

**Comment by pronist** [comment]
@julyrunt 스크롤 바를 체크해야겠네요. 좋은 개선점 감사드립니다.
